prov/rxm/src/rxm_cq.c

```diff
@@ -130,6 +156,9 @@ static int rxm_finish_send_nobuf(struct rxm_tx_entry *tx_entry)
 					"Unable to report completion\n");
 			return ret;
 		}
+#if ENABLE_DEBUG
+		rxm_cq_log_comp(tx_entry->comp_flags);
+#endif
```

Comment:
nit: a cleaner way of doing this is to move the ENABLE_DEBUG around the call definition, and provide an empty macro in the else case.